lib/cryptoexchange/exchanges/bitmart/services/market.rb

```diff
@@ -0,0 +1,37 @@
+module Cryptoexchange::Exchanges
+  module Bitmart
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          url = ticker_url(market_pair)
+          output = super(url)
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Bitmart::Market::API_URL}/ticker/#{market_pair.base}_#{market_pair.target}"
+        end
+
+        def adapt(output, market_pair)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Bitmart::Market::NAME
+          ticker.ask       = NumericHelper.to_d(output['ask_1'])
+          ticker.bid       = NumericHelper.to_d(output['bid_1'])
+          ticker.last      = NumericHelper.to_d(output['new_24h'])
+          ticker.volume    = NumericHelper.to_d(output['volume'])
+          ticker.timestamp = output['openTime']
```

Comment:
output['closeTime'] is more appropriate, also time format has to be in UNIX format. 
the output is 1531729086853, so please do `output['closeTime'] / 1000`